Makefile

```diff
@@ -52,7 +52,11 @@ console:
 	./_build/$(PROFILE)/rel/blockchain_node/bin/blockchain_node remote_console
 
 docker-build:
-	docker build -t helium/node .
+	docker build \
```

Comment:
1. are you forgetting genesis block here?
2. make docker-start does not appear to work?

Reply:
1. It should get included through the Dockerfile instructions.
2. That needs to be addressed. :|